#sqr point3d
obst_avoid/line_point3d[] sqr_p3d
